staging/src/github.com/openshift/oc/pkg/cli/admin/release/mirror.go

```diff
@@ -442,6 +451,26 @@ func (o *MirrorOptions) Run() error {
 	} else {
 		fmt.Fprintf(o.Out, "\nSuccess\nUpdate image:  %s\nMirrored to: %s\n", to, o.To)
 	}
+	
+	mirrorRef, _ := imagereference.Parse(o.To)
+	mirrorRepo := mirrorRef.AsRepository().String()
+
+	sourceRef, _ := imagereference.Parse(o.From)
+	sourceRepo := sourceRef.AsRepository().String()
+
+	fmt.Fprintf(o.Out, "\nTo use the new mirrored release-image to install, provide the following list of release-image sources to the installer using the install-config.yaml:\n")
+	fmt.Fprintf(o.Out, `
+imageContentSource:
+- sources:
```

Comment:
Is it better to create a internal struct that maps matches the output and use that to marshal yaml.. string manipulation is always great.

Reply:
I was not pleased with doing it this way.  I had considered creating a yaml type structure but since we want a specific order I just when with strings to maintain desired order.  Could you provide an example of the internal struct you are suggesting?

Reply:
somethings like this.
```go
type installConfigSubsection struct{
    ImageContentSources []installConfigSubsectionImageContentSource `json:"imageContentSources"`
}
type installConfigSubsectionImageContentSource struct {
    Sources []string `json:"sources"`
}
```

you can still control the order in the Sources list...?